File: coding/18th March 2025/Palindrome Number.py
#Palindrome Number
class Solution:
    def isPalindrome(self, n: int) -> bool:
        if n < 0:
            return False
        res = 0
        n1 = n
        while n > 0:
            res = res * 10 + n % 10
            n = n // 10
        return res == n1

File: coding/17th December 2025/Palindrome Number.java
//Palindrome Number
class Solution 
{
    public boolean isPalindrome(int n) 
    {
        int res = 0;
        int n1 = n;
        while(n > 0)
        {
            int r = n % 10;
            res = res * 10 + r;
            n = n / 10;
        }
        return res == n1;
    }
}
